.env.example

```diff
@@ -8,7 +8,7 @@ APP_NAME=UBTV
 APP_ENV=local
 APP_KEY=
 APP_DEBUG=true
-APP_URL=http://localhost
+APP_URL=http://localhost:8000
```

Comment:
is this changes necessary? may i know the reason why you changed it in the first place

Reply:
I updated the APP_URL to http://localhost:8000 in the .env.example for simplicity and to match the default Laravel development server URL. This can help avoid potential issues related to CORS or URL mismatches during local development.

Reply:
Okay, that make sense. Thanks for the reply